mw-webapp/src/logic/redirectPage/RedirectPage.module.scss

```diff
@@ -0,0 +1,10 @@
+@import "src/styles/variables.scss";
+@import "src/styles/mixins.scss";
+
+.redirectPageContainer {
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
```

Comment:
global!!

Reply:
what?

Reply:
Value of margin top to global variable

Reply:
Should I put it in a separate variable?

Reply:
Put it inside _variables.scss, redirect-page-margin or something like this